### PR TITLE
docs: add latest changes to release notes and add a few extra note to upgrade guide

### PR DESCRIPTION
**Pull request by nicolethoen**
All changes merged into main since v5 has been merged have not been included in the release notes table. I updated the release notes table to include (only breaking - not bug fx) changes merged since v5 has been renamed to main.

I also added a couple notes to the upgrade guide after getting some notes back from @jpuzz0 doing OCM migration testing.

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 1]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
...
+  {
+    component: "Charts",
+    description: (<>Fixed dark theme chart var prefix from <code className='ws-code'>pf-v5-c-chart</code> to <code className='ws-code'>pf-v5-chart</code>.</>),
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5673",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Fonts",
+    description: "Fixed RHD bold incorrect font mapping to italic font.",
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:4967]:
```suggestion
    description: "Fixed incorrect font mapping from bold to italic for RHD font.",
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 2]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
...
+    pullRequestURL: "https://github.com/patternfly/patternfly/pull/5678",
+    repo: "HTML/CSS",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Text input group",
+    description: "Addressed the cause of the earch input icon and placeholder text being the wrong color when in a masthead or dark block.",
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:4974]:
```suggestion
    description: "Addressed the cause of the search input icon and placeholder text being the wrong color when in a masthead or dark block.",
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 3]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
...
+  {
+    component: "Popover",
+    description: (<>Reverted default value of popover <code className='ws-code'>appendTo</code> prop so it is now <code className='ws-code'>() => document.body</code>.</>),
+    pullRequestURL: "https://github.com/patternfly/patternfly-react/pull/9252",
+    repo: "React",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Toolbar",
+    description: (<>Allows <code className='ws-code'>alignItems</code> and <code className='ws-code'>alignSelf</code> on <code className='ws-code'>ToolbarContent</code>, <code className='ws-code'>ToolbarGroup</code> and <code className='ws-code'>ToolbarItem</code> to have values <code className='ws-code'>start</code>, <code className='ws-code'>center</code>, and <code className='ws-code'>baseline</code>.</>),
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:5030]:
```suggestion
    description: (<>Updated <code className='ws-code'>alignItems</code> and <code className='ws-code'>alignSelf</code> on <code className='ws-code'>ToolbarContent</code>, <code className='ws-code'>ToolbarGroup</code> and <code className='ws-code'>ToolbarItem</code> to have values <code className='ws-code'>start</code>, <code className='ws-code'>center</code>, and <code className='ws-code'>baseline</code>.</>),
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 4]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
     repo: "react-user-feedback",
     fixedWithCodeMod: false
   },
+  {
+    component: "Form control",
+    description: "Changes all the divs in a form control to be spans to allow a form control to be inside phrasing content.",
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:4946]:
```suggestion
    description: "Changed all the divs in a form control to be spans to allow a form control to be inside phrasing content.",
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 5]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
...
+  {
+    component: "Select",
+    description: (<>Used text input component in the typeahead toggle for the deprecated select.</>),
+    pullRequestURL: "https://github.com/patternfly/patternfly-react/pull/9317",
+    repo: "React",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "@All",
+    description: (<>Create component level package.json files to enable federated modules.</>),
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:5079]:
```suggestion
    description: (<>Created component level package.json files to enable federated modules.</>),
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js [hunk 6]
```diff
@@ -4941,4 +4941,158 @@ import { Wizard } from '@patternfly/react-core/deprecated';`}
...
+    pullRequestURL: ["https://github.com/patternfly/patternfly-react/pull/9287","https://github.com/patternfly/patternfly-react/pull/9260", "https://github.com/patternfly/patternfly-react/pull/9272"],
+    repo: "React",
+    fixedWithCodeMod: false
+  },
+  {
+    component: "Charts",
+    description: (<>Update Victory packages to the latest version (36.6.11).</>),
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-notes-data.js:5086]:
```suggestion
    description: (<>Updated Victory packages to the latest version (36.6.11).</>),
```

File: packages/documentation-site/patternfly-docs/content/get-started/upgrade.md [hunk 7]
```diff
@@ -92,3 +92,8 @@ PatternFly 5 brings a new implementation to the following components, which can
 - [Select](/components/menus/select)
 - [Dropdown ](/components/menus/dropdown) 
 - [Wizard](/components/wizard)
+
+## Other notes
+
+ - When updating the react-topology package, note that the topology styles have been removed from their previous home in @patternfly/react-styles and now need to be imported directly from react-topology.
```
[the remark this answers is not in the document]
Comment (nicolethoen) [reply packages/documentation-site/patternfly-docs/content/get-started/upgrade.md:98]: I don't think this is so much a change. We just discovered that if people were importing that way (some project were, but they shouldn't have been) then it definitely won't work anymore.

File: packages/documentation-site/patternfly-docs/content/get-started/upgrade.md [hunk 8]
```diff
@@ -92,3 +92,8 @@ PatternFly 5 brings a new implementation to the following components, which can
...
+ - In general, though PatternFly ships a `src` directory in it's npm package, it is not advised to import components from a `src` directory. Instead, when importing via absolute paths, import components from `dist` directories.
```
[the remark this answers is not in the document]
Comment (nicolethoen) [reply packages/documentation-site/patternfly-docs/content/get-started/upgrade.md:99]: I can add some code snippets :)

**Review by edonehoo — DISMISSED** [review]
🚗

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 9]
```diff
@@ -125,16 +125,16 @@ As PatternFly has matured, a number of its popular components have been repeated
...
-| [Table (deprecated)](/components/table/react-deprecated)                        | [Table](/components/table)                                                 |
-| [Wizard (deprecated)](/components/wizard/react-deprecated)                      | [Wizard](/components/wizard)                                               |
+| Deprecated component                                                            | Recommended replacement                |
+|---------------------------------------------------------------------------------|----------------------------------------|
+| [Application launcher](/components/menus/application-launcher/react-deprecated) | [Dropdown](/components/menus/dropdown) |
+| [Context selector](/components/menus/context-selector/react-deprecated)         | [Dropdown](/components/menus/dropdown) |
+| [Dropdown (deprecated)](/components/menus/dropdown/react-deprecated)            | [Dropdown](/components/menus/dropdown) |
+| [Options menu](/components/menus/options-menu/react-deprecated)                 | [select](/components/menus/select)     |
```
Comment (evwilkin) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:133]: `Select` should be uppercase here
```suggestion
| [Options menu](/components/menus/options-menu/react-deprecated)                 | [Select](/components/menus/select)     |
```

**Comment by evwilkin** [comment]
@nicolethoen one nit about uppercasing the Select component name in the deprecated/recommended table, but it's not blocking and otherwise LGTM

**Review by kmcfaul — APPROVED** [review]
lgtm

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by tlabaj — APPROVED** [review]
LGTM other than PR that went in today not being listed.

**Pull request merged**